README.md

```diff
@@ -1,27 +1,55 @@
 # Template for projects using map4sci
 
+## Introduction
+
+The `map4sci project template` is a template that provides an abstraction layer for running Map4sci, making it easy to integrate Map4sci into new projects. It helps developers to add more scripts and create new repositories using the same template, which simplifies the development process and makes it easy to maintain the codebase.
+
+To use the map4sci project template, simply clone the template repository using `git clone https://github.com/cns-iu/map4sci-project-template.git` command and use it as a starting point for your own project. The template includes all the necessary files and configuration to run Map4sci and includes a run.sh script that can be used to start the Map4sci application.
```

Comment:
Actually, this is a GitHub Template repo (https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-template-repository), they can create a new repo based on this template by going here: https://github.com/cns-iu/map4sci-project-template/generate

Reply:
Still this ☝️